pydis_site/apps/api/tests/test_reminders.py

```diff
@@ -54,6 +54,18 @@ def setUpTestData(cls):
             name='Mermaid Man',
             discriminator=1234,
         )
+        cls.user = User.objects.create(
+            id=5678,
+            name='Fish Dude',
+            discriminator=5678,
+        )
+        cls.role = Role.objects.create(
+            id=555,
+            name="Random role",
+            colour=2,
+            permissions=0b01010010101,
+            position=10,
+        )
```

Comment:
It isn't necessary to create these objects since there's FK being used. Just use some integer literals for the IDs in the test.